src/components/SnackbarContext.tsx

```diff
@@ -37,7 +37,7 @@ export const SnackbarProvider: React.FC<SnackbarProviderProps> = ({
       {children}
       <Snackbar
         open={open}
-        autoHideDuration={6000}
+        autoHideDuration={3000}
```

Comment:
we should use variables in the code and assign the values in the variables.

Reply:
I've refactored the code to use variables for better readability and maintainability. Now, all the values are assigned to variables. I've tested the changes to ensure that they work.